daemon/config.go

```diff
@@ -44,6 +44,7 @@ type Config struct {
 	Context                     map[string][]string
 	TrustKeyPath                string
 	Labels                      []string
+	EnableHairpinNAT            bool
```

Comment:
I used `HairpinNAT` everywhere else, but in this structure there's already `EnableIPv6`, `EnableIPv6`, `EnableIpForward`, `EnableIpMasq`, `DisableNetwork`, `EnableCors`,  `EnableSelinuxSupport `, ... so I figured I'd be consistent :p